### ISSUE TITLE
https://identifiers.org/ gives a blank page in a QtWebKit-based application

### ISSUE THREAD

**Issue by agarny** [identifiers-org/identifiers-org.github.io#92]
Hi, [OpenCOR](https://opencor.ws/) has been using identifiers.org for quite some time, but it's only now that we are looking into supporting the new platform. Basically, we use identifiers.org to render the web page about an ontological repository (e.g. https://registry.identifiers.org/registry/go and https://registry.identifiers.org/registry/chebi) or an ontological term (e.g. https://identifiers.org/GO:0032237 and https://identifiers.org/CHEBI:38807).

OpenCOR relies on [QtWebKit](https://github.com/qt/qtwebkit) to render web pages and we have never had any problems with it until today.

However, none of the identifiers.org links for an ontological repository gets rendered properly. We only get the message about cookies. For example, here is what we get for https://registry.identifiers.org/registry/go:

![Screen Shot 2020-04-03 at 7 10 50 PM](https://user-images.githubusercontent.com/602265/78329724-ed247380-75de-11ea-87f1-ae2a1417cd74.png)

while an identifiers.org link for an ontological term will get rendered as expected (since it redirects to a different URL). For example, here is what we get for https://identifiers.org/GO:0032237:

![Screen Shot 2020-04-03 at 7 13 36 PM](https://user-images.githubusercontent.com/602265/78329894-3bd20d80-75df-11ea-83c4-b8d8fed2de39.png)

Basically, https://identifiers.org/ renders a blank page and so does anything that starts with https://registry.identifiers.org/. The only thing from identifiers.org that renders fine is https://docs.identifiers.org/. (FWIW, http://ebi.identifiers.org/ is dead.)

If we "inspect" the page, we can see that 1.8 MB has been downloaded:

![Screen Shot 2020-04-03 at 7 39 40 PM](https://user-images.githubusercontent.com/602265/78331631-f7487100-75e2-11ea-8ad4-7cb1dd5f853e.png)

![Screen Shot 2020-04-03 at 7 39 50 PM](https://user-images.githubusercontent.com/602265/78331663-02030600-75e3-11ea-91ec-fb918afa7d5c.png)

So, any idea what might be going wrong here?

Cheers, Alan.

**Comment by agarny** [comment]
For what it is worth, I have (using Google Chrome) saved https://identifiers.org/ as a complete web page (i.e. including the JavaScript, CSS, etc. files) and loaded that page locally in my QtWebKit-based application and it's all working fine.

So, it looks like there is indeed nothing wrong with the site itself. Yet, I am getting a blank page when trying to load https://identifiers.org/...

**Comment by mbdebian** [comment]
Dear @agarny ,
I'd like to thank you for submitting this request, specially with this level of detail.
Unfortunately, at identifiers.org we are not familiar with QTWebKit, and, in this case, we can't really get beyond making sure our web frontends are up and running and there's been no outage.
I'm sorry we cannot be of more help with QTWebKit, please, don't hesitate to get back to us in case you have any other question.
Kind Regards,
Manuel

P.S.: If I had to make a guess, I would check how QTWebKit deals with iFrames, as, although being a very old technique, and it should not be used any more, at identifiers.org we use Keycloak as an authentication and identity provider, and it comes with a connecting Javascript library that uses iFrames for some kind of side communication with the Auth server.

**Comment by agarny** [comment]
Just for the record, the problem is due to a bug in old WebKit, which is what QtWebKit uses (see [here](https://forum.qt.io/topic/113418/blank-page-as-a-result-of-using-iframes/)).
